content/pages/privacy/en.md

```diff
@@ -1,152 +1,153 @@
 
 **Legal basis:**
 
 The legal basis for this processing is our legitimate interest, according to Art. 6(1)(f) GDPR.
 
 **Legitimate interests:**
 
-Our legitimate interest is to enhance the user experience by improving the performance of our market.
+Our legitimate interest is to enhance the user experience by improving the performance of our portal demonstrator and to enable visitors to find and consume your data service.
 
 **Retention period:**
 
 Your wallet address and name are stored permanently on our metadata cache.
 
 ## 4.5 History table
 
-The history table of our market is a transparency tool that you can use to overview your transactions relating to our market. When you use the history table, we collect your wallet address. Then we retrieve the respective and relevant transactions stored on the blockchain (more precisely, from the metadata cache) and organize them in a table.
+The history table of our portal demonstrator is a transparency tool that you can use to overview your transactions relating to our portal demonstrator. When you use the history table, we collect your wallet address. Then we retrieve the respective and relevant transactions stored on the Gaia-X testnet (more precisely, from the metadata cache) and organize them in a table.
 
 **Purpose:**
 
-We need to collect your wallet address to retrieve every transaction you made on our market. We retrieve and organize your public transaction data so you can quickly overview all the actions you made on our market.
+We need to collect your wallet address to retrieve every transaction you made on our portal demonstrator. We retrieve and organize your public transaction data so you can quickly overview all the actions you made on our portal demonstrator.
 
 **Legal basis:**
 
 The legal basis for this processing is our legitimate interest, according to Art. 6(1)(f) GDPR.
 
 **Legitimate interests:**
 
-Our legitimate interest in providing the history table is to offer you the possibility to overview your transactions made on our market.
+Our legitimate interest in providing the history table is to offer you the possibility to overview your transactions made on our portal demonstrator.
 
 **Retention period:**
 
 As soon as you disconnect your wallet, we will remove the history table entries from your browser.
 
 ## 4.6 Contact via e-mail
 
-If you contact us via e-mail, deltaDAO will collect, use and store your e-mail address, and any other information you provide us is your message, such as your name.
+If you contact us via e-mail, Gaia-X AISBL will collect, use and store your e-mail address, and any other information you provide us is your message, such as your name.
```

Comment:
issue(blocking): Change `... information you provide us is your message ...` to `... information you provide us in your message ...`